### PR TITLE
Update docs 2.8.2

### PR DESCRIPTION
**Pull request by salvacorts**
**What this PR does / why we need it**:

Update docs and changelog with Loki 2.8.2 changes.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by vlad-diachenko — APPROVED** [review]
LGTM

**Comment by salvacorts** [comment]
Halting until we get a confirmation on Slack

File: CHANGELOG.md [hunk 1]
```diff
@@ -80,6 +75,18 @@
...
+#### Loki
+
+##### Security
+
+* [9370](https://github.com/grafana/loki/pull/9370) **dannykopping**: upgrade to go1.20.4
+
+##### Enhancements
+
+* [9184](https://github.com/grafana/loki/pull/9184) **periklis**: Bump dskit to introduce IPv6 support for memberlist
+
```
Comment (DylanGuedes) [inline CHANGELOG.md:89]:
```suggestion

#### Promtail

##### Enhancements

* [8994](https://github.com/grafana/loki/pull/8994) **DylanGuedes**: Promtail: Add new `decompression` configuration to customize the decompressor behavior.
```

**Review by DylanGuedes — CHANGES_REQUESTED** [review]
LGTM, but could you also include this change and remove it from the unreleased section? I backported it because users on 2.8 were getting crashes when decompressing without this

File: CHANGELOG.md (hunk 1 above)
Comment (salvacorts) [reply CHANGELOG.md:89]: So that commit is already in the `release-2.8.x` branch? Or should it be added?

Comment (DylanGuedes) [reply CHANGELOG.md:89]: it is already present on release-2.8.x

Comment (salvacorts) [reply CHANGELOG.md:89]: Done

**Pull request merged**